foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -21,6 +22,20 @@
 #include "websocket_notls.hpp"
 #include "websocket_tls.hpp"
 
+// Debounce a function call (tied to the line number)
+// This macro takes in a function, the debounce time in milliseconds, and any
+// arguments to pass to the function.
+#define FOXGLOVE_DEBOUNCE(f, ms, ...)                                                     \
+  do {                                                                                    \
+    static auto last_call_##__LINE__ = std::chrono::system_clock::now();                  \
```

Comment:
I don't think `__LINE__` is really necessary since the variable is scoped within the do{}while() anyway.

Reply:
Without `__LINE__`, if you use the macro in different places, they would interfere since they would use the same static variable

Reply:
https://godbolt.org/z/MWasY7on7

Reply:
@achim-k that would be true if this was a function, but it's a macro. Each instance of `FOXGLOVE_DEBOUNCE` will copy-paste a new variable definition inside a `do { }` block.

Reply:
Oh, you are right! Will change it